src/Components/ProductFilters.jsx

```diff
@@ -1,12 +1,39 @@
-import { Disclosure, Menu, Transition } from "@headlessui/react";
-import { ChevronDownIcon, FilterIcon } from "@heroicons/react/solid";
-import React, { Fragment } from "react";
+import { Disclosure, Menu, Transition } from '@headlessui/react';
+import { ChevronDownIcon, FilterIcon } from '@heroicons/react/solid';
+import React, { Fragment, useEffect } from 'react';
+import { useState } from 'react';
 
 function classNames(...classes) {
-  return classes.filter(Boolean).join(" ");
+  return classes.filter(Boolean).join(' ');
 }
 
-export default function ProductFilters({ filterOptions, setFilterOptions, sortOptions, setSortOptions }) {
+export default function ProductFilters({
+  filterOptions,
+  setFilterOptions,
+  sortOptions,
+  setSortOptions,
+  products,
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** This is quite a lot of props coming in to this component. This usually indicates a smell. Check out [how I fixed the functionality of this component here](https://github.com/developer-job-simulation/react-ecommerce/pull/59/files#diff-677d130aa04e6e50d7d277ed352ffae135a674dde48bcf5e44f09f50e910ae2bR27).‏